### DISCUSSION TITLE
Question about SCPH calculation in ANPHON for higher order cases

### DISCUSSION THREAD

**Discussion by Gi-HoonKang** [ttadano/alamode#89] [Q&A] [answered]
Dear Dr.Tadano,

Hello. 

Actually, I asked some questions via e-mail before. It was very helpful to me. 
First of all, I would say that I appreciate your careful answer.

Anyway, I have a question related to SCPH calculation process in ANPHON for higher order cases.
I obtained force constants including sixth order by using "ALM". As far as I remember, you also used the force constant file including sixth order to do SCPH calculation in ALAMODE tutorial session. However, I concerned whether SCPH calculation can include sixth order terms or not. According to the theoretical background page and your research paper, expansion up to fourth terms was described.

In summary, would SCPH calculation be still valid for higher order  (more than fourth term) cases?

Best regards,
Gi-Hoon Kang

**Comment by ttadano** [comment, marked as answer]
Yes, your understanding is correct. 

While the sixth-order IFCs can be included in the fitting process of ALM, they are not used in the SCPH mode of ANPHON; Instead, only fourth-order IFCs will be used.

This is the limitation of current implementation, which may be lifted in a future version of alamode.
